src/autogluon_dashboard/app.py

```diff
@@ -48,8 +48,12 @@
 from autogluon_dashboard.scripts.widget import Widget
 
 # TODO: Remove hardcoded default csv path
-dataset_file = os.environ.get("PER_DATASET_S3_PATH", "dev_data/all_data.csv")
-aggregated_file = os.environ.get("AGG_DATASET_S3_PATH", "dev_data/autogluon.csv")
+dataset_file = os.environ.get(
+    "PER_DATASET_S3_PATH", "https://dashboard-test-yash.s3.us-west-2.amazonaws.com/dev_data/all_data.csv"
+)
+aggregated_file = os.environ.get(
+    "AGG_DATASET_S3_PATH", "https://dashboard-test-yash.s3.us-west-2.amazonaws.com/dev_data/autogluon.csv"
+)
```

Comment:
So these couple lines are constantly changing between PRs.

Reply:
Yeah I keep forgetting to change them back to the s3 url. I use the local path for testing